tests/quote.py

```diff
@@ -0,0 +1,57 @@
+import conftest
+
+class Quote:
+    @staticmethod
+    def choose_error_tolerance(amount_in: int):
```

Comment:
nit: we call it on amount_out as well. Let's rename to a more general `amount`?

Reply:
Indeed, good catch!